runner.py

```diff
@@ -1,138 +1,228 @@
-                             "By default uses 115200")
-
-    parser.add_argument("--no-flash",
-                        default=False, action='store_true',
-                        help="Board will not be flashed by runner.")
-
-    # Add alternative option "--long-test" "" for long-test argument, which is False
+    parser.add_argument(
+        "-T",
+        "--target",
+        default="ia32-generic-qemu",
+        choices=targets.keys(),
+        help="Set target on which tests will be run. "
+        "By default runs tests on %(default)s target. "
+        "Targets can be extended using extension feature.",
+    )
+
+    parser.add_argument(
+        "-H",
+        "--host",
+        default="pc",
+        choices=hosts.keys(),
+        help="Set host that run tests. \"pc\" host is recommended for running tests on emulators. "
+        "To control physical boards you can use \"rpi\" host or pc "
+        "(but it requires restarting board manually by yourself). By default runs tests on %(default)s. "
+        "Hosts can be extended using extension feature.",
+    )
+
+    parser.add_argument(
+        "-t",
+        "--test",
+        default=[],
+        action="append",
+        type=args_file,
+        help="Specify directory in which test will be searched. "
+        "If flag is not used then runner searches for tests in "
+        "phoenix-rtos-project directory. Flag can be used multiple times.",
+    )
+
+    parser.add_argument(
+        "-p",
+        "--port",
+        help="Specify serial to communicate with device board. "
+        "Default value depends on the target",
+    )
+
+    parser.add_argument(
+        "-b",
+        "--baudrate",
+        default=115200,
+        help="Specify the connection speed of serial. " "By default uses %(default)d",
+    )
+
+    parser.add_argument(
+        "--no-flash",
+        default=False,
+        action="store_true",
+        help="Board will not be flashed by runner.",
+    )
+
+    parser.add_argument(
+        "--no-test",
+        default=False,
+        action="store_true",
+        help="Tests will be not run by runner. This option may be helpful to only flash the device "
+        "or check if yamls are parsed correctly."
+    )
+
+    parser.add_argument(
+        "-v",
+        "--verbose",
+        action="count",
+        default=0,
+        help="Increases verbosity level",
+    )
+
+    # Add alternative option "--nightly" "" for nightly argument, which is False
     # Needed for proper disabling long tests in gh actions
     # because of problem with passing empty argument through gh workflows
-    parser.add_argument("--long-test",
-                        type=bool,
-                        nargs='?', default=False, const=True,
-                        help="Long tests will be run")
+    parser.add_argument(
+        "--nightly",
+        type=bool,
+        nargs="?",
+        default=False,
+        const=True,
+        help="Nightly tests will be run",
+    )
```

Comment:
I prefered LONG_TEST name but if this sticks we should also change build env param

Reply:
For now this variable is intended only for our ports' test suites, that usually take more time. I see that `nightly` in this context may be better, but when someone wants to build ports with tests setting  `NIGHTLY` variable to `y` could be misleading. I would prefer to leave it as it is and underline in documentation that nightly runner campaigns run long tests (currently test suites for ports), use power off restart if possible and so on.